src/__main__.py

```diff
@@ -1,44 +1,27 @@
 import asyncio
 import os
 import textwrap
-from itertools import cycle
 
 import discord
 from discord.ext import commands
 from discord.ext.commands import Bot
 
-from cogs.utils import constants
+from utils import config
+envs = config.envs
```

Comment:
```suggestion
from utils import config


envs = config.envs
```
`flake8` standards requite 2 lines gap between imports and the rest of the code